### DISCUSSION TITLE
Can GitHub Copilot Solve LeetCode Problems?

### DISCUSSION THREAD

**Discussion by whoami-anoint** [community/community#21144] [Copilot Conversations]
Can GitHub Copilot Solve LeetCode Problems?

**Comment by devkennyy** [comment]
Mostly, yes. Refer to [this video](https://www.youtube.com/watch?v=FHwnrYm0mNc) for an entertaining demonstration.

**Comment by ZikunWang24** [comment]
I just developed a chrome extension called LeetCopilot to use generative AI to help people learn Leetcode problems easily right on the page, without needing to switch tabs. It offers context-aware hints, auto-generated study notes and mock interviews. Feel free to try it out :) https://leetcopilot.dev/
